frontend-ssr/pages/submissions.module.scss

```diff
@@ -0,0 +1,31 @@
+@import "../styles/variables";
+.page-submissions {
+  width: $width;
+  max-width: $max-width;
+  form {
```

Comment:
Uhmm...there's no form in the rendered elements? 😅

Reply:
Yes..because I also did the search component. It should be in Search.module.scss. I thought it should be no problem if I let the code here for now, but I will remove it.

Reply:
Then let's add the search component as part of this PR as well.

Reply:
oky

Reply:
Sorry, I didn't realize it yesterday, but if it's a separate component, it should have separate styles in it's own `.module.scss` file.